CHANGES.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## 1.10.15 (core) / 0.26.15 (libraries)
+
+### New
+
+- Added a config section to `dagster.yaml` to enable submitting backfill runs in a threadpool.
+- Expanded definition time validation for partition mappings to avoid runtime errors querying asset status.
```

Comment:
[ui][beta] You can now re-execute a run that targeted a multi-asset from the point of _asset_ failure instead of _step_ failure, meaning only assets that failed or were skipped will be re-executed. To enable this option, turn on the `Enable retries from asset failure` feature flag in your user settings.